src/modules/claim/services/claim.service.ts

```diff
@@ -269,6 +269,35 @@ export class ClaimService {
     return qb.getMany();
   }
 
+  /**
+   * Get claims able to be revoked by user with matching DID
+   * @param filters additional filters
+   * @param currentUser revoker's DID
+   */
+  async getByRevoker({
+    currentUser,
+    filters: { namespace } = {},
+  }: {
+    filters?: QueryFilters;
+    currentUser: string;
+  }) {
+    let allRolesByRevoker = await this.rolesByRevoker(currentUser);
+    if (namespace) {
+      allRolesByRevoker = allRolesByRevoker?.filter(
+        (role) => role.namespace === namespace
+      );
+    }
+    const rolesByRevoker = allRolesByRevoker?.map((r) => r.namespace);
```

Comment:
```suggestion
    const rolesByRevoker = allRolesByRevoker.map((r) => r.namespace);
```

Reply:
Happy to change, but just curious what the reservation is about the undefined or null check?